docs/streams/developer-guide/dsl-api.html

```diff
@@ -2969,6 +2971,62 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
+			    <span id="windowing-final-results"></span><h5><a class="toc-backref" href="#id31">Window Final Results</a><a class="headerlink" href="#window-final-results" title="Permalink to this headline"></a></h5>
+			    <p>In Kafka Streams, windowed computations update continuously.
+			       As new data arrives for a window, freshly computed results are emitted downstream.
+			       For most applications, this is ideal, since fresh results are always available with a minimum of configuration,
+			       and Kafka Streams is designed to make programming continuous computations seamless.
+			       However, some applications need to take action <strong>only</strong> on the final result of a windowed computation.
+			       A common example of this is sending alerts.
+			    </p>
+			    <p>Suppose that you have an hourly windowed count of events per user.
+			       If you wish to send an alert when a user has more than three events in an hour, you would need some state tracking
+			       to make sure that you don't send two alerts when a user has four events in one hour.
+			       This is a little cumbersome, but doable.
+			    </p>
+			    <p>On the other hand, if you want to send an alert when a user has <em>less than</em> three events in an hour, you have a real challange.
+			       This is fundamentally because all users would match this condition at first, until they accrue enough events, so you cannot simply
+			       send an alert when someone matches the condition; you have to wait until you know you won't see any more events for a particular window
+			       and <em>then</em> send the alert.
+			    </p>
+                            <p>Kafka Streams offers a cleaner way to support both of these use cases: after defining your windowed computation, you can
+			       <span class="pre">suppress</span> the intermediate results, emitting the final count for each user when the window <strong>closes</strong>.
+			    </p>
+			    <p>For example:</p>
+			    <div class="highlight-java"><div class="highlight">
+<pre style="line-height: 100%;font-family:monospace;background-color:#ffffff; border-width:0.01mm; border-color:#000000; border-style:solid;padding:4px;font-size:18pt;">
+KGroupedStream&lt;UserId, Event&gt; grouped = ...;
+grouped
+    .windowedBy(TimeWindows.of(Duration.ofHours(1)).grace(ofMinutes(10)))
+    .count()
+    .suppress(Suppressed.untilWindowCloses(unbounded()))
+    .filter((windowedUserId, count) -&gt; count &lt; 3)
+    .toStream()
+    .foreach((windowedUserId, count) -&gt; sendAlert(windowedUserId.window(), windowedUserId.key(), count));
+</pre>
+			    </div></div>
+			    <p>The key parts of this program are:
+			    <dl>
+				    <dt><pre>.windowedBy(...grace(ofMinutes(10)))</pre></dt>
+				    <dd>This allows us to bound the lateness of events the window will accept.
+				        For example, the 09:00 to 10:00 window will accept late-arriving records until 10:10, at which point, the window <em>closes</em>.
```

Comment:
nit: `the window is closed` -- a window is something passive and doesn't close itself IMHO